packages/rum-angular/test/polyfills.ts

```diff
@@ -28,30 +28,6 @@
  * https://angular.io/guide/browser-support#polyfills-for-non-cli-users
  */
 
-/*
- * IE9, IE10 and IE11 requires all of the following polyfills.
- */
-import 'core-js/features/symbol'
-import 'core-js/features/object'
-import 'core-js/features/function'
-import 'core-js/features/parse-int'
-import 'core-js/features/parse-float'
-import 'core-js/features/number'
-import 'core-js/features/math'
-import 'core-js/features/string'
-import 'core-js/features/date'
-import 'core-js/features/array'
-import 'core-js/features/regexp'
-import 'core-js/features/map'
-import 'core-js/features/weak-map'
-import 'core-js/features/set'
-import 'core-js/features/reflect'
-
-/*
- * Evergreen browsers require these
- */
-import 'core-js/proposals/reflect-metadata'
```

Comment:
Is the `reflect-metadata` no longer needed? 
Is type metadata stored in a different way?

Reply:
the angular `ng` build system adds their own polyfills based on `browserlist` in package.json. I will update this with correct list in the upcoming E2E PR.